Update jquery.ui.combify.js

Yarli opened the pull request:
Added the ability to cap the maximum length of the input field.

Yarli committed:
Update jquery.ui.combify.js

Added the ability to cap the maximum length of the input field.

Yarli committed:
Update README.md

Yarli committed:
Update jquery.ui.combify.js

steelheaddigital commented:
Thanks for this.  When I try to use your new option, maxLength, I'm getting the following error:

Uncaught ReferenceError: inputSelector is not defined

It looks like the variable inputSelector is not defined anywhere.  Maybe you wanted to use textInputSelector?  Also, would the mind adding a simple test for this to the file in /test/UITest.html?

Thanks again

Yarli committed:
Update jquery.ui.combify.js

Amended textInputSelector from inputSelector. Looks like I picked this up from an old fork of this project.

Yarli committed:
Update UITest.html

Added example of maxLength

Yarli commented:
Sorry about that. I think when I added that modification I was working on an old copy of the project on my system here. I only copied over the relevant code from my copy into the new branch and I didn't notice the change. Sorry about that. 

I've now updated my mistake :) and also added an example into the requested file too. 

Kind regards

Adam French
QueByte Network Services

Mobile: 07747 778581
Phone: 0845 0041866

This email was sent from my iPhone so it's likely that I am out of the office. 


> On 14 Jan 2015, at 04:02, Tom Mooney <notifications@github.com> wrote:
> 
> Thanks for this. When I try to use your new option, maxLength, I'm getting the following error:
> 
> Uncaught ReferenceError: inputSelector is not defined
> 
> It looks like the variable inputSelector is not defined anywhere. Maybe you wanted to use textInputSelector? Also, would the mind adding a simple test for this to the file in /test/UITest.html?
> 
> Thanks again
> 
> �\
> Reply to this email directly or view it on GitHub.
>

steelheaddigital commented:
Looks good.  I have now merged it.  Thanks a lot.
